libs/community/langchain_community/chat_models/baichuan.py

```diff
@@ -113,14 +121,16 @@ def lc_secrets(self) -> Dict[str, str]:
     def lc_serializable(self) -> bool:
         return True
 
-    baichuan_api_base: str = Field(default=DEFAULT_API_BASE)
+    baichuan_api_base: str = Field(default=DEFAULT_API_BASE, alias="base_url")
     """Baichuan custom endpoints"""
     baichuan_api_key: SecretStr = Field(alias="api_key")
     """Baichuan API Key"""
     baichuan_secret_key: Optional[SecretStr] = None
     """[DEPRECATED, keeping it for for backward compatibility] Baichuan Secret Key"""
     streaming: bool = False
     """Whether to stream the results or not."""
+    max_tokens: int = 2048
```

Comment:
would this introduce a new token limit by default?

Reply:
Yes, the default value for max tokens introduced in the latest documentation is 2048.

Reply:
What is the benefit of maintaining that default here? If it changes Langchain will have an incorrect default. Does it make sense for this to be `Optional[int] = None`?

Reply:
Wow! I understand, I will improve it.